### PR TITLE
og for post

### PR DESCRIPTION
**Pull request by wingfeatherwave**
Screenshot 2024-02-02 at 9 37 33 AM

### CODE REVIEW

File: api/og/postimage/[id].tsx [hunk 1]
```diff
@@ -0,0 +1,158 @@
+// @ts-nocheck
+
+import React from 'react';
+
+import type { VercelRequest } from '@vercel/node';
+import { ImageResponse } from '@vercel/og';
+
```
Comment (crabsinger) [inline api/og/postimage/[id].tsx:7]: Looks like ReactMarkdown sits on top of a https://github.com/remarkjs/remark/tree/main/packages/remark remark unified pipeline.

So I think we may need to use `remark` directly to render Markdown outside React, however I'm not sure what the exact details of the remark pipeline options are, and we'll want to figure this out to ensure that browser and og markdown is identical.

Comment (wingfeatherwave) [reply api/og/postimage/[id].tsx:7]: not going to render md at this time

**Comment by crabsinger** [comment]
sounds good - lets ship this as is

**Comment by crabsinger** [comment]
@wingfeatherwave needs a rebase - might be a bit gnarly.. I'd recommend a `git merge main`

**Review by crabsinger — DISMISSED** [review]
@wingfeatherwave code seems good and ready to merge.

Only issue is the text in the modal doesn't match the current implementation.
Screenshot 2024-02-12 at 1 45 16 PM

Since the og tags will render but you wont be able to see it logged out in browser...

**Review by crabsinger — APPROVED** [review]
@wingfeatherwave you can merge if this is ready

**Pull request merged**